src/XrdPfc/XrdPfcFile.hh

```diff
@@ -208,7 +208,7 @@ public:
    // Constructor, destructor, Open() and Close() are private.
 
    //! Static constructor that also does Open. Returns null ptr if Open fails.
-   static File* FileOpen(const std::string &path, long long offset, long long fileSize);
+   static File* FileOpen(const std::string &path, long long offset, long long fileSize, XrdOucCacheIO* inputIO);
```

Comment:
This actually changes ABI of `libXrdPfc-5.so`. However, since it's a plugin and normally used with a matching version of XRootD, I don't see a problem with merging into a normal feature release.